### DISCUSSION TITLE
Disable Guided Learning UI Element

### DISCUSSION THREAD

**Discussion by JasonFosterNS** [burkybang/SuiteAdvanced-Feedback#153] [Ideas]
It would be beneficial to have a setting to enable/disable the guided learning UI Element.

**Comment by burkybang** [comment]
Since you closed the discussion, I assume you found the existing setting 😃

**Comment by JasonFosterNS** [comment]
LOL. Yes I did!  As always, SuiteAdvanced is amazing.

Get Outlook for iOS<https://aka.ms/o0ukef>

Jason Foster
He/Him/His (Learn more here)
Manager
Enterprise Applications
Global Consulting Solutions
Tel: 959-200-7195
Fax: 959-200-7300
***@***.***



CohnReznick LLP
The information contained herein (or in any attachment) is not intended to be used by any taxpayer for the purpose of avoiding any penalties that a taxing authority might impose on the taxpayer or for the promoting, marketing or recommending to another party any tax related matters.

The information in this transmission is privileged and confidential and intended only for the recipient listed above. If you are not the intended recipient, please advise the sender immediately by reply e-mail and delete this message and any attachments without retaining a copy. If you are not the intended recipient, you are hereby notified that any disclosure, copying or distribution of this message, or the taking of any action based upon it, is strictly prohibited.  
________________________________
